vendor/github.com/minio/minio/pkg/mimedb/Makefile

```diff
@@ -0,0 +1,17 @@
+# Generate db.go from db.json downloaded nodejs mime-db project.
+# NOTE: Autogenerated db.go needs to be vet proofed. \
+	Manually edit json -> JSON for all variable names
```

Comment:
don't need to vendorize this, ignore this from `git reset HEAD^ vendor/github.com/minio/minio/pkg/mimedb/Makefile` and ammend.

Reply:
done